cdc/sink/manager_test.go

```diff
@@ -188,17 +188,19 @@ func (s *managerSuite) TestManagerAddRemoveTable(c *check.C) {
 		}
 	}
 
+	redoManager := redo.NewDisabledManager()
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
 		// add three table and then remote one table
-		for i := 0; i < goroutineNum; i++ {
+		for i := 0; i < 200; i++ {
```

Comment:
Why don't we keep using it?

Reply:
revert it, maybe caused by merge conflict.